src/utils/primaryKey-is-ForeignKey.js

```diff
@@ -0,0 +1,3 @@
+exports.primaryKeyIsForeignKey = (association) =>
+  Object.values(association.source.rawAttributes).filter((attr) =>
+    attr.field === association.source.primaryKeyField).length > 1;
```

Comment:
[PROPOSAL] Knowing this will be the only one function of the this file, we might want to export it as it is, and packed in an object.
`module.exports = (association) => ..`